### PR TITLE
Added script to store daily stats in kmq db

### PR DESCRIPTION
**Pull request by taahamahdi**
I'll setup the daily cronjob to call this directly on prod

### CODE REVIEW

File: src/scripts/store-daily-stats.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import Knex from "knex";
+import kmqKnexConfig from "../config/knexfile_kmq";
+
+const storeDailyStats = async () => {
+    const db = Knex(kmqKnexConfig);
+
+    if (!(await db.schema.hasTable("daily_stats"))) {
```
Comment (Brainicism) [inline src/scripts/store-daily-stats.ts:7]: This should be a migration

File: src/scripts/store-daily-stats.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        await db.schema.createTable("daily_stats", (table) => {
+            table.date("date").notNullable();
```
Comment (Brainicism) [inline src/scripts/store-daily-stats.ts:9]: should this be unique?

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: src/scripts/store-daily-stats.ts (hunk 1 above)
Comment (taahamahdi) [reply src/scripts/store-daily-stats.ts:7]: Your documentation in DEVELOPER.md was very helpful, thanks!

File: src/scripts/store-daily-stats.ts (hunk 2 above)
Comment (taahamahdi) [reply src/scripts/store-daily-stats.ts:9]: Yup, thanks

File: migrations/20201123155842_store-daily-stats.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+exports.up = function(knex) {
+    return knex.schema.createTable("daily_stats", (table) => {
+        table.date("date").notNullable();
```
Comment (Brainicism) [inline migrations/20201123155842_store-daily-stats.js:3]: should this be unique

**Review by Brainicism — CHANGES_REQUESTED** [review]

File: migrations/20201123155842_store-daily-stats.js (hunk 3 above)
Comment (taahamahdi) [reply migrations/20201123155842_store-daily-stats.js:3]: I marked it as unique below. Originally I added unique on the same line, but I followed how you did it in another file

**Review by Brainicism — APPROVED** [review]

**Pull request merged**